include/nuttx/fs/fs.h

```diff
@@ -222,6 +222,10 @@ struct geometry
   bool      geo_writeenabled; /* true: It is okay to write to this device */
   blkcnt_t  geo_nsectors;     /* Number of sectors on the device */
   blksize_t geo_sectorsize;   /* Size of one sector */
+
+  /* NULL-terminated string representing the device model */
+
+  char      geo_model[NAME_MAX + 1];
```

Comment:
Same comment for the MTD applies for the handlers of `BIOC_GEOMETRY`.

Reply:
Done.